scarlet/assets/forms.py

```diff
@@ -25,6 +25,9 @@ class Meta(object):
     def __init__(self, *args, **kwargs):
         super(UploadAssetForm, self).__init__(*args, **kwargs)
         self.fields["tags"].widget.attrs["class"] = "widget-tags"
+        self.fields["tags"].label = "Search Tags"
```

Comment:
These shouldn't be hard coded like this in the front end. The backend model field for Tags should take in a verbose_name and help_text fields. It looks like you'll have to do that on the TaggableManager (https://github.com/markmiscavage/scarlet/blob/master/scarlet/cms/internal_tags/taggit_handler.py) by overriding this https://github.com/jazzband/django-taggit/blob/master/taggit/managers.py#L422

Reply:
Ok, I'll change it, I thought that AutoTagModel is more generic solution thats why I've changed on the front end part

Reply:
fixed, moved to AutoTagModel